src/Fable.React.Native.fsproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Version>2.6.1</Version>
     <PackageVersion>2.6.1</PackageVersion>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
I would keep the netstandars2 target for now to avoid issues with library consumers. Just by referencing FSharp.Core 5 and using dotnet SDK 5 you should have access to latest F# features.